.github/scripts/end2end/generate-kustomization.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Constants for valid topologies for CI tests
+# We support multiple shards per host, or one shard per host, up to 9 nodes
+readonly VALID_TOPOLOGIES=(
+    "1:1" "1:2" "3:1" "3:3" "6:1" "6:3" "6:2" "6:6" "9:1" "9:3" "9:6" "9:9"
```

Comment:
- no "3:2" and "9:2" (seem to be missing from the logical series..)
- should be sorted : `"6:1" "6:3" "6:2"` --> `"6:1" "6:2" "6:3"`

Reply:
> ...but the other shard count (2, 3, 6) seem arbitrary to me, we could do any kind of "striping" but probably cn't really support these cases at the moment?

Can you be more specific on the ones you don't understand?
As 6 nodes 2 shards is fine for example, as this is what we want to support.

Reply:
> As 6 nodes 2 shards is fine for example, as this is what we want to support.

indeed

> Can you be more specific on the ones you don't understand?

* 6:3 : why have 2 instances/replicas per rs?
* 9:6 : how does it map 6 shards to 9 nodes?
* 3:2 and 9:2 I suggested earlier : same, how does that map 2 shards to 3 and 9 nodes, respectively?